src/shell/dotnet/copy-fdc3-iife-bundle.ps1

```diff
@@ -0,0 +1,10 @@
+# Copying the IIFE bundle of FDC3 to the shell' Preload folder
+# so that it can be preloaded for the Embedded Browser
+
+
+$bundle = "./src/shell/js/composeui-fdc3/dist/fdc3-iife-bundle.js"
```

Comment:
This will only work from the repo root. You can use `$PSScriptRoot` to build the path from the directory where the script is located.